tests/playbooks/tests_dummy.yml

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: dummy0
+    profile: "{{ interface }}"
+    lsr_fail_debug:
+      - __network_connections_result
+  tasks:
+    - debug:
+        msg: "this is: playbooks/tests_dummy.yml"
+      tags:
+        - always
+
+    - block:
+        - include_tasks: tasks/run_test.yml
+          vars:
+            lsr_description: Create a dummy interface
+            lsr_setup:
+              - tasks/delete_interface.yml
+              - tasks/assert_device_absent.yml
+            lsr_test:
+              - tasks/create_dummy_profile.yml
+            lsr_assert:
+              - tasks/assert_profile_present.yml
+              - tasks/assert_device_present.yml
+            lsr_cleanup:
+              - tasks/cleanup_profile+device.yml
+      tags:
+        - "tests::cleanup"
```

Comment:
cleanup seems to be wrong here

Reply:
got it. I found the root cause of the failure. 

`nmcli con add type dummy ifname dummy0 ipv4.addresses 192.0.2.42/30` will never create a profile in `/etc/sysconfig/network-scripts/`

Maybe we can change the tests file to exclude `- tasks/assert_profile_present.yml`